free_thresh: 0.196
gps_origin: [52.341726, 4.915569]
image: amsterdam_cropped.png
map_bottom: 5798303
map_left: 622457
map_right: 632457
map_top: 5810803
negate: 0
occupied_thresh: 0.65
origin: [-500.0, -500.0, 0.0]
resolution: 0.5
